File: dsclient/lib/Packets/BaseRequest.dart
class BaseRequest{
    late String token;
    late int sequence;
    late String url;

    BaseRequest(String token, int sequence, String url){
        this.token = token;
        this.sequence = sequence;
        this.url = url;
    }

    Map<String, dynamic> toJson() => {
        'token': token,
        'sequence': sequence,
        'url': url
    };
}

File: DsPacket/Packets/BaseRequest.cs
﻿namespace DsPackets
{
    public class BaseRequest
    {
        public string token { get; set; } = string.Empty;
        public int sequence { get; set; } = 0;
        public string url { get; set; }

        public BaseRequest(string val)
        {
            url = val;
        }
    }
}
